RPIPs/RPIP-49.md

```diff
@@ -56,41 +72,15 @@ We'd love to get team feedback here, especially on ballpark timelines.
   - Dramatically improve capital efficiency 
 
 ### Scenario D (2 releases):
-- Release 1: Megapools, Remove operator-gated upgrades
+- Release 1: Megapools, Forced delegaet upgrades
   - Allows ETH-only participants
   - Allows NOs to migrate to 4-ETH minipools for greater efficiency
```

Comment:
Change language to match Scenario B:

    Allows NOs to move to Megapools for greater efficiency
    Allow 4-ETH validators within Megapools

Avoids confusion compared to "migrating to 4-ETH minipools"